contrib/examples/deploy.sh

```diff
@@ -2,7 +2,12 @@
 oc create namespace cluster-operator
 
 # Create ssl certs for api server
-./contrib/apiserver-aggregation-tls-setup.sh
+if [ ! -e apiserver.pem ]
+then
+	./contrib/apiserver-aggregation-tls-setup.sh
+else
+	echo "Re-using existing certs."
```

Comment:
What is the benefit of re-using the certs? Is this a step towards making deploy.sh idempotent?

Reply:
It appeared that controller manager would get SSL errors if you re-ran with new certs, that might be only if you have modified the controller manager deployment yaml and not the apiserver though.

Reply:
Ah, I see. Yeah, restarting one of the deployments but not the other would cause problems. Is re-running deploy.sh going to work in that case any more after the changes from #8?

Should we just pull the cert creation script out entirely and require it be run separately at some point in the past?

Reply:
I think I may be ignorant about exactly the behavior of openshift templates. Does `oc apply` return 0 when the template would create resources that already exist? So that the `oc apply` part essentially skips over the apiserver deployment and only attempts to create the controller-manager deployment? If so, is it ignoring that the parameters given to the template have been changed?

Reply:
I can't see any issues with #8 at a glance, will test today.

Feels ok to let it decide if it should gen or not based on if they exist. (my check is a bit rudimentary right now but still) Little smoother onboarding new folks, one less pre-development step. I wouldn't mind nesting them under a certs/ dir though so they're easy to blow away.

Reply:
Regarding apply, my suspicion is that it updates the secret correctly with the new cert, but there's nothing that will automatically redeploy when a secret changed, so as far as kube is concerned the apiserver deployment is still the same.